packages/foreman/foreman-fapolicyd/foreman-fapolicyd.spec

```diff
@@ -0,0 +1,94 @@
+Name:     foreman-fapolicyd
+Version:  1.0.0
+Release:  1%{?dist}
+Summary:  Foreman fapolicyd rules
+
+Group:    System Environment/Base
+License:  GPLv3+
+URL:      https://theforeman.org
+Source0:  60-foreman.rules
+Source1:  61-foreman-proxy.rules
+
+BuildArch: noarch
+
+Requires: fapolicyd
+Requires: foreman-proxy-fapolicyd
+
+%description
+Foreman fapolicyd rules
+
+%prep
+
+%build
+
+%install
+
+mkdir -p %{buildroot}%{_sysconfdir}/fapolicyd/rules.d
+cp %{SOURCE0} %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/60-foreman.rules
+cp %{SOURCE1} %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/61-foreman-proxy.rules
+
+%post
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+  %{_bindir}/systemctl restart fapolicyd.service
+else
+  %{_sbindir}/fagenrules --load
+fi
+
+%postun
+
+rm -f %{buildroot}%{_sysconfdir}/fapolicyd/rules.d/60-foreman.rules
+
+if %{_bindir}/systemctl is-active --quiet fapolicyd.service; then
+  %{_sbindir}/fagenrules --load
+  # loading fagenrules leaves fapolicyd.service in a dead state, restart to ensure
+  %{_bindir}/systemctl restart fapolicyd.service
+else
+  %{_sbindir}/fagenrules --load
+fi
+
+%files
+%defattr(-,root,root)
+%attr(0644,root,fapolicyd) %{_sysconfdir}/fapolicyd/rules.d/60-foreman.rules
```

Comment:
probably a note for later, but IMHO such files belong to `/usr/share/fapolicyd` or `/usr/lib/fapolicyd`, not `/etc`.

Reply:
It was a memo to you and me to talk to fapolicyd team whether it does support and lobby for if it does not. ;)